training/distributed_training/tensorflow/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Add a Cleanup section that deletes the estimator and any other resources created in the notebook.</p><p> </p><p>Add a Conclusion section that summarizes what the user accomplished/learned and any recommended next steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>